api/src/main/java/net/wesjd/anvilgui/AnvilGUI.java

```diff
@@ -122,6 +126,46 @@ private static ItemStack itemNotNull(ItemStack stack) {
     /**
      * Create an AnvilGUI
      *
+     * @param plugin              A {@link org.bukkit.plugin.java.JavaPlugin} instance
+     * @param player              The {@link Player} to open the inventory for
+     * @param mainThreadExecutor  An {@link Executor} that executes on the main server thread
+     * @param titleComponent      What to have the text already set to
+     * @param initialContents     The initial contents of the inventory
+     * @param preventClose        Whether to prevent the inventory from closing
+     * @param geyserCompatibility Whether to enable compatibility with Geyser software
+     * @param closeListener       A {@link Consumer} when the inventory closes
+     * @param concurrentClickHandlerExecution Flag to allow concurrent execution of the click handler
+     * @param clickHandler        A {@link ClickHandler} that is called when the player clicks a slot
+     */
+    private AnvilGUI(
```

Comment:
there is no need to add a new constructor here, because the constructor is private and only used in the builder. you can safely just edit the old constructor.